src/index.html

```diff
@@ -11,8 +11,52 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute should point to a CSS file instead of an SCSS file. Consider changing `href="./styles/index.scss"` to `href="./styles/index.css"` to ensure the stylesheet is correctly applied.